bootstrap.php

```diff
@@ -5,3 +5,26 @@
 use Retrofit\Drupal\Provider;
 
 $GLOBALS['conf']['container_service_providers']['retrofit'] = Provider::class;
+
+
+spl_autoload_register(function (string $item) {
+    if (!\Drupal::hasContainer()) {
+        return null;
+    }
+    $infoParser = new \Drupal\Core\Extension\InfoParser(\Drupal::root());
+    $moduleList = \Drupal::moduleHandler()->getModuleList();
+    foreach ($moduleList as $module) {
+        $info = $infoParser->parse($module->getPathname());
+        $files = $info['files'] ?? [];
```

Comment:
One optimization is to save the modules which have `files` keys as a container parameter during container compilation.